src/bindings/js/node/tests/infer_request.test.js

```diff
@@ -6,31 +6,32 @@ const ov = require('../build/Release/ov_node_addon.node');
 const assert = require('assert');
 const { describe, it } = require('node:test');
 const { getModelPath } = require('./utils.js');
+const util = require('node:util');
+
+const testXml = getModelPath();
+const core = new ov.Core();
+const model = core.readModel(testXml);
+const compiledModel = core.compileModel(model, 'CPU');
+
+const inferRequest = compiledModel.createInferRequest();
+
+const tensorData = Float32Array.from({ length: 3072 }, () => Math.floor(Math.random() * 3072));
```

Comment:
Array length is 3072. But why do array elements have limitation of max value limit of 3072? :)

Reply:
Assertions from assert library might be failing for very small numbers. I added a task to consider adding `Chai Assertion Library` or try different solution

Reply:
My question was not about array length. This is fine and thank you for your attention with assertion behavior.
Elements of `tensorData` array will be numbers from 0 to 3072. It is just interesting for me, why did you choose the same value that array length has? It's little bit confusing because somebody may think that these values (array length and max array value) are depended. But they don't)

Reply:
I changed this value to `epsilon`